config/ca_cert.yaml

```diff
@@ -4,17 +4,40 @@
 #@ load("@ytt:data", "data")
 #@ load("@ytt:overlay", "overlay")
 
+#@ def split_certs():
+#@   ca_certs_lines=data.values.ca_cert_data.splitlines()
+#@   ca_certs = []
+#@   i1=0
+#@   i2=0
+#@   for ca_cert_line in ca_certs_lines:
+#@     if ca_cert_line == "-----END CERTIFICATE-----" :
+#@       ca_certs.append("\n".join(ca_certs_lines[i1:i2+1]))
+#@       i1=i2+1
+#@     end
+#@     i2+=1
+#@   end
+#@   return ca_certs
+#@ end
+
+#@ ca_certs=split_certs()
+
 #@ if data.values.ca_cert_data != "":
+#@ i = 0
+#@ for ca_cert in ca_certs:
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: ca-certificates-binding
+  name: #@ "ca-certificates-binding-" + str(i)
```

Comment:
Should this be a versioned resource? https://carvel.dev/kapp/docs/v0.46.0/diff/#versioned-resources